packetbeat/main.go

```diff
@@ -5,6 +5,17 @@ import (
 
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/packetbeat/beater"
+
+	// import support protocol modules
+	_ "github.com/elastic/beats/packetbeat/protos/amqp"
```

Comment:
We should automate this in the future. Also on my list for metricbeat, so perhaps we will be able to use the same script.